### ISSUE TITLE
Implementing nother algorithm

### ISSUE THREAD

**Issue by jaykodeveloper** [aleju/self-driving-truck#6]
Hi Aleju,

It was really great work!! I want to implement my own calculation, but don't know where to start. If my code is reinforcement, can I just change code in train.py in train_reiforced directory?

Thanks,

**Comment by aleju** [comment]
Yes, I think all the reinforcement learning stuff is in `train_reinforced/`. The reward computation is in `lib/rewards.py`, possible actions in `lib/actions.py`, replay memory saving and loading in `lib/replay_memory.py` and the reading and communication with the game window are mainly in `lib/ets2game.py` and `lib/ets2window.py`.
Won't be a trivial task to change the algorithm. Scrolling through the files, it seems like I did not exactly aim for the highest code quality.
